src/integration/integration_api_metricscollector_test.go

```diff
@@ -53,6 +54,22 @@ var _ = Describe("Integration_Api_MetricsCollector", func() {
 		stopMetricsCollector()
 	})
 	Describe("Get metrics", func() {
+
+		Context("Check user authorization failed", func() {
```

Comment:
check permission not passed 

you may add another test case for  cloud controller/uaa  api not available